mobileApp/src/redux/selectors.ts

```diff
@@ -0,0 +1,25 @@
+import {createSelector} from 'reselect';
+import {IInitState} from './reducers';
+
+const initState = (state: IInitState) => state;
+
+export const selectorHistory = createSelector(
```

Comment:
Please, use reselect selectors only for selectors that compute some value from state. In other cases you should write simple clean function.